src/elements/Button/ButtonGroup.js

```diff
@@ -0,0 +1,79 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import META from '../../utils/Meta'
+import * as sui from '../../utils/semanticUtils'
+import {
+  getUnhandledProps,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+/**
+ * Button.Group
+ */
+function ButtonGroup(props) {
+  const {
+    basic, className, children, color, icon, size, vertical, widths,
+  } = props
+
+  const classes = cx('sd-button-group', 'ui',
```

Comment:
Heads up, I've finished removing all `sd-*` classes, see #301.  Rebase to master to get the update.  The test for the `sd-*` class has been inverted to fail if any `sd-` class exists.

Let's remove all `sd-*` classes from all PRs.  👍